test/runtests.jl

```diff
@@ -63,3 +63,5 @@ for name in (
         include("test_$name.jl")
     end
 end
+
+Plots.set_backend!(; force = true)  # clear out `Preferences` file
```

Comment:
Does that mean that everyone who runs the tests will have his preferences wiped?

Reply:
It depends how they run the tests. If they use `Pkg.test`, then no. That creates a new temporary environment where the LocalPreferences.jl file will be stored.

Reply:
So currently, a package is imported in `Main` (hack for not adding it as a hard dependency in `Project.toml`). However, for some packages, you need to access the module itself (e.g. https://github.com/JuliaPlots/Plots.jl/blob/4fff117f5587abe30361ec367346752f10c5cbe6/src/backends/pyplot.jl#L12). We import `GR` directly in plots, but other backends need this. And for consistency, I've added this for all backends (here plotly).

Reply:
Should be fixed by https://github.com/JuliaPlots/Plots.jl/pull/4566/commits/c920c49662a17d3a046aaf111019f304f2b6cdd0.